src/app/pages/rpc-send-transfer/components/send-transfer-actions.layout.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactNode } from 'react';
+
+import { Box } from '@stacks/ui';
+
+export function SendTransferActionsLayout(props: { children: ReactNode }) {
+  const { children } = props;
+  return (
+    <Box bottom="0px" position="absolute" px="loose" width="100%">
```

Comment:
Why do we need to set 0 here? Is this to override the use of `Stack` higher up?

Reply:
I was pushing the action buttons to the bottom.